ember-cli-build.js

```diff
@@ -75,6 +75,18 @@ module.exports = function () {
     },
     'ember-power-select': {
       theme: false
+    },
+    postcssOptions: {
+      compile: {
+        enabled: false
+      },
+      filter: {
+        enabled: true,
+        exclude: ['vendor/**/*'],
+        plugins: [
+          require('tailwindcss')('./app/styles/app/tailwind/config.js')
```

Comment:
Would it make sense to put Tailwind to `./app/styles/tailwind`, to separate it from our old styles? I also thought about `./config/tailwind.js`, but this will require to restart ember serve after each change. What do you think?

Reply:
I like the idea of separating it more from our old styles. :+1: 

And yeah, it feels weird to have the config file outside the `config` folder, and especially in the `styles` folder. But I also don't like the idea of needing to restart ember serve after a change to it. It could go by itself in some other folder, like `/app/tailwind/config.js`, which I've seen elsewhere, but that feels weird too. Or maybe `/app/tailwind.config.js`? I'm open to different approaches on this

Reply:
I think it should belong to `/app/styles` as it basically defines styles of the app. All our current styles live in `/app/styles/app/**`, so I though, it would be good to have Tailwind styles in `/app/styles/tailwind/**`. Although, this is just my theoretical suggestion and you may have some sensible objections.

Reply:
Nah, no objections here. That makes sense to me :smile: